openbb_platform/platform/core/openbb_core/app/model/system_settings.py

```diff
@@ -25,15 +25,15 @@ class SystemSettings(Tagged):
     platform: str = FrozenField(default=str(pl.platform()))
 
     # OpenBB section
-    # TODO: Get the version of the SDK from somewhere that's not pyproject.toml
+    # TODO: Get the version of the Platform from somewhere that's not pyproject.toml
     version: str = FrozenField(default=VERSION)
     home_directory: str = FrozenField(default=str(HOME_DIRECTORY))
     openbb_directory: str = FrozenField(default=str(OPENBB_DIRECTORY))
     user_settings_path: str = FrozenField(default=str(USER_SETTINGS_PATH))
     system_settings_path: str = FrozenField(default=str(SYSTEM_SETTINGS_PATH))
 
     # Logging section
-    logging_app_name: Literal["sdk"] = FrozenField(default="sdk")
+    logging_app_name: Literal["platform"] = FrozenField(default="platform")
```

Comment:
@hjoaquim is this fine?

Reply:
Yes, no issues here. Posthog will just get the new value.